scripts/person/person.gd

```diff
@@ -420,8 +420,19 @@ func getessence():
 	###---Expansion End---###
 	return essence
 
-
-
+#ralph
+func playercleartraits():
```

Comment:
This function implements balance changes without condtion, specifically the clearing of _mod stats. Though I would prefer that this function was removed from here and mainmenu.gd in favor of altering cleartraits() to perform the change dependent on self == globals.player and tweaks enabled.

Reply:
Gotcha, I was looking at this as I was going down the comments from mainmenu. I'll follow this idea.

Reply:
Fixed